Makefile

```diff
@@ -91,6 +91,8 @@ help:
 	@echo "        Create an RPM repository with modules"
 	@echo "    fixtures/rpm-with-modules-modified"
 	@echo "        Create an RPM repository with modules but without some dependencies."
+	@echo "    fixtures/rpm-modular"
+	@echo "        Create an RPM repository with modules, module defaults and module obsoletes."
```

Comment:
Perhaps we could talk with them about having `static-context` covered as well?  I don't see any modules using it in here.

Reply:
I adjusted [gen-patched-fixtures.sh](https://github.com/pulp/pulp-fixtures/pull/230/files#diff-bb6476db87bc921265dfdb517b7ee0e36009a5b36c98086e68d7dfc2f619825f) and patch for ``static-context`` as well that we can test this field with new fixtures.